demo-app/src/open-truss/configs/4-trino-demo.yaml

```diff
@@ -12,16 +21,28 @@ workflow:
             SELECT name, phone, acctbal, comment
             FROM tpch.sf1.customer
             limit 10
-          results: :results
+          # output: [:resultsArray, :resultsObject, :name]
```

Comment:
I left this here to show what I am going to do as a follow up. I'm going to modify the renderer to pass in signals into objects and arrays as well as update the `OTUquiDataProvider` component to use the signals found in `outputs`. This will result in deleting `resultsArray` and `resultsObject` below.